pkg/controller/node/controller_utils.go

```diff
@@ -223,9 +228,14 @@ func markAllPodsNotReady(kubeClient clientset.Interface, nodeName string) error
 		if pod.Spec.NodeName != nodeName {
 			continue
 		}
-
+		isForgive := isPodForgivable(pod, nodeStatus.readyTransitionTimestamp)
```

Comment:
markAllPodsNotReady will be called only if node status transition to "not ready"